packages/familie-backend/src/index.ts

```diff
@@ -28,6 +28,7 @@ class Backend {
         sessionKonfigurasjon: ISessionKonfigurasjon,
         saksbehandlerTokenConfig: ITokenRequest,
         prometheusTellere?: { [key: string]: Counter },
+        msGraphOBOAccessTokenConfig?: ITokenRequest,
```

Comment:
I dont think we need this as a parameter. It can be configured in the package and just used. It will be useful for all our apps.

Reply:
ok